.github/workflows/build.yml

```diff
@@ -1,42 +1,36 @@
 name: Tests
+
 on:
   push:
-    branches:
-      - main
   pull_request:
-    branches:
-      - main
+    paths-ignore:
+      - "CONTRIBUTING.md"
```

Comment:
Let's not do this. It doesn't play well with required status checks (this will stop the workflow from running if the only file changed is CONTRIBUTING.md, which sounds like it saves work since we know the tests can't possibly fail in that case, but the PR can't be landed because the required status checks will never pass)